### ISSUE TITLE
How to add a zoom indicator to the address bar?

### ISSUE THREAD

**Issue by paluch-stanislaw** [Feodor2/Mypal#253]
Hi, how to add a zoom indicator to the address bar?

![Screenshot_2](https://user-images.githubusercontent.com/69870350/94466089-2a9a4000-01c1-11eb-97ec-c43def72f674.png)

**Comment by Feodor2** [comment]
With the addon, look only in the old firefox addons.

**Comment by paluch-stanislaw** [comment]
Previously i'm used Firefox 52.9.0 ESR and it was.

Screenshot without any addons:


![Screenshot_1](https://user-images.githubusercontent.com/69870350/94860693-b36de180-0436-11eb-9c16-5b760f44a1ae.png)

**Comment by Feodor2** [comment]
Yes, though mypal based on the older firefox 28 interface before this feature was added, there is a centaury browser which has firefox 52 interface including zoom indicator.
